components/post/create/Input.tsx

```diff
@@ -0,0 +1,43 @@
+import styled from '@emotion/styled';
+import { useState, ChangeEventHandler, ChangeEvent } from 'react';
+
+interface Props {
+  type?: string;
+  id: string;
+  placeholder: string;
+  width?: number;
+  height?: number;
+}
+
+function Input({ type, id, placeholder, width, height }: Props) {
+  const [value, setValue] = useState('');
+
+  const handleValueChange: ChangeEventHandler = (e: ChangeEvent<HTMLInputElement>) => {
+    setValue(e.target.value);
+  };
+
+  return (
+    <StyledInput
+      type={type}
+      id={id}
+      value={value}
+      placeholder={placeholder}
+      width={width}
+      height={height}
+      autoComplete="off"
+      onChange={handleValueChange}
+    />
+  );
+}
+
+const StyledInput = styled.input((props) => {
+  return {
+    outline: 'none',
+    paddingLeft: '10px',
+    marginBottom: '20px',
+    width: props.width,
+    height: props.height,
+  };
+});
```

Comment:
p4  이런 식으로 작성해도 좋을 것 같아요~

```suggestion
const StyledInput = styled.input(({width, height}) => {
  return {
    outline: 'none',
    paddingLeft: '10px',
    marginBottom: '20px',
    width: width,
    height: height,
  };
});
```

Reply:
p4

더욱 간결한 방식 제안합니다~
```suggestion
const StyledInput = styled.input(({width, height}) => {
  return {
    outline: 'none',
    paddingLeft: '10px',
    marginBottom: '20px',
    width,
    height,
  };
});
```

Reply:
ㅎㅎ.. 사실 props에서 가져오고 있다는걸 좀 표현하고 싶어서 이렇게 썼었는데 수정해보겠습니다~!!